src/Services/AuthService.ts

```diff
@@ -0,0 +1,20 @@
+import * as jwt from "jsonwebtoken";
+import {ServiceError} from "./ServiceError";
+import {JwtPayload} from "../Types/JwtPayloadType";
+import {NodeService} from "./NodeService";
+import config from "../Config/Config";
+
+export class AuthService {
+
+    public async authorizerUser(token: string) {
+        const authorizedUser = await jwt.verify(token, config.jwtKey);
```

Comment:
`const authorizedUser: JwtPayload = await jwt.verify(token, config.jwtKey); `

is something that should work too for typing and maybe easier to read